Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/EventBased/ChatAnalytics.cs

```diff
@@ -38,5 +40,18 @@ private void OnChatBubbleVisibilityChanged(bool isVisible)
                 { "is_visible", isVisible },
             });
         }
+
+        private void OnConversationClosed()
+        {
+            analytics.Track(AnalyticsEvents.UI.CHAT_CONVERSATION_CLOSED);
+        }
+
+        private void OnConversationOpened(bool wasAlreadyOpen)
+        {
+            analytics.Track(AnalyticsEvents.UI.CHAT_CONVERSATION_OPENED, new JsonObject
+            {
+                { "was_already_open", wasAlreadyOpen },
+            });
```

Comment:
Can we instead assemble the string we will send over the Internet directly and avoid some allocations?